### ISSUE TITLE
To many labels in the quickstarts

### ISSUE THREAD

**Issue by davsclaus** [fabric8io/ipaas-quickstarts#748]
When you deploy a quickstart such as the camel-servlet it has 5 or more labels in the kube overview, and 2 labels has the same value

![screen shot 2015-10-01 at 2 16 04 pm](https://cloud.githubusercontent.com/assets/477100/10220315/b4cce7f6-6846-11e5-9270-af215580736f.png)

We should maybe try for the fabric8:json goal to not generate so many labels.
Also the version is now a label. I guess maybe this is on purpose?

**Comment by davsclaus** [comment]
notice that component and project has the same label value

**Comment by jstrachan** [comment]
yeah - thats my fault; I recently added a 'project' annotation by default (and a 'version') so that we can more easily query the RCs to show which project versions are deployed in an environment in the Environments tab.

So we should retire the use of the 'component' annotation really

**Comment by davsclaus** [comment]
Okay as they are defined in the quickstarts we can remove it form those pom.xml files.

**Comment by jimmidyson** [comment]
so what would this look like? i use `component` label all the time to clean stuff up - would i swap to using `project`?

**Comment by davsclaus** [comment]
Yeah component is in many today also in the devops

https://github.com/fabric8io/fabric8-devops/blob/master/cdelivery/pom.xml#L67

**Comment by jstrachan** [comment]
@jimmidyson yeah - I figured 'project' was a bit more descriptive; its meant to be the project name (or BuildConfig name in OpenShift) (which defaults to the artifact id by default)
